Merging changes for travic ci plugin

raror3 opened the pull request:
Merging changes for travic ci plugin for building with specific jdk version - 8 and also to create zip file distribution - as a pre-requisite for travis to upload release tag post building.

raror3 committed:
Configuring to build with jdk8

Configuring to build with jdk8

raror3 committed:
Merge pull request #1 from raror3/raghav/travis_java_version

Configuring to build with jdk8

raror3 committed:
Adding plugin parameter to create zip

Adding plugin parameter to create zip of distribution as part of build step.

raror3 committed:
Adding Travis CI badge

Adding Travis CI badge for Spring XD build.

sapient-support commented on .travis.yml:
```diff
@@ -14,6 +17,6 @@ env:
 
 script:
-  - travis_wait ./gradlew --debug clean build -PtestLoggingStarted=true -PwikiBranch=origin/master
+  - travis_wait ./gradlew --debug clean build distZip -PtestLoggingStarted=true -PwikiBranch=origin/master
```
This will just generate zip file. how do we upload to GIT HUB

raror3 committed:
Updated readMe.adoc

raror3 committed:
Update README.md

raror3 committed:
Added markdown badge for travis ci

raror3 committed:
Updated markdown badge for travis ci

raror3 committed:
Disabling tests while building

Disabling tests while building

raror3 committed:
Skipping the installation step 

Skipping the installation step to not let Travis execute the default build script and thus run out customized build script.
